.gitignore

```diff
@@ -10,4 +10,6 @@
 /Gemfile.lock
 
 # Files generated by update script
-examples_with_updated_last_modified.yml
+_data/examples_with_updated_last_modified.yml
+_data/examples_status.yml
+_data/examples_with_updated_dates.yml
```

Comment:
I don't think we use `examples_with_updated_dates.yml`, it was the old name of the "last_modified" yaml, if memory serves ?

Reply:
Ok. Then I will just merge this now.

Reply:
(Removed `examples_with_updated_dates.yml`.)